src/react-apps/package-lock.json

```diff
@@ -5188,7 +5188,7 @@
 		},
 		"duplexer": {
 			"version": "0.1.1",
-			"resolved": "https://registry.npmjs.org/duplexer/-/duplexer-0.1.1.tgz",
+			"resolved": "http://registry.npmjs.org/duplexer/-/duplexer-0.1.1.tgz",
```

Comment:
Why change react package-lock. when you only have changed C# code?

Reply:
Didn't note this. i have to remove this file. I shouldn't have checked in this